Gemfile

```diff
@@ -108,4 +105,6 @@ group :test do
   # for test coverage report
   gem 'simplecov', require: false
 
+  # for controller testing
+  gem 'rails-controller-testing'
```

Comment:
As mentioned in the upgrading guide - 

`assigns` and `assert_template` have been extracted to the `rails-controller-testing` gem 

http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html